recipes/magic-cta-pipe/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "magic-cta-pipe" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/magic_cta_pipe-{{ version }}.tar.gz
+  sha256: 341b96234744b0f3807b6fd9089b179d3c65c69bf4bd8ab27c5870f78fa132ef
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=64
```

Comment:
in MCP there is no lower limit, just setuptools<=71 in environment (and nothing in setup file)